### ISSUE TITLE
Is there documentation on how to configure s3?

### ISSUE THREAD

**Issue by kstaken** [terascope/file-assets#229]
I don't see any information on how to configure teraslice to actually use S3.

**Comment by godber** [comment]
Yeah the connector is not documented at all and it's requirements just changed.  Lets find an appropriate place to include these docs.

**Comment by godber** [comment]
Adding minio to the docker compose would make sense too.

**Comment by godber** [comment]
There is a docker compose example and a config that needs to be updated under this directory:

https://github.com/terascope/teraslice/blob/master/examples/es-to-s3/

**Comment by busma13** [comment]
The readme for file-assets contains s3 connector configuration info.
